metadata/collect_model_metadata.py

```diff
@@ -20,8 +20,43 @@ def robust_asdict(obj):
         return obj
 
 
+def config_data_to_features(config_data: dict) -> dict:
+    """
+    Reformat fields from config.json to match our features.
+    """
+    # TODO: None - has to be filled in manually/I'm not sure what field it corresponds to
```

Comment:
I'm not sure if some of these things are in config files, for the ones with None I didn't see corresponding fields in an example config, let me know if there's a way to fill them out @neubig , @lindiatjuatja ?